osu.Game.Tests/Visual/UserInterface/TestSceneNowPlayingOverlay.cs

```diff
@@ -49,9 +49,10 @@ public void TestPrevTrackBehavior()
             AddStep(@"Call PrevTrack", () => musicController.PrevTrack());
             AddAssert(@"Check if it restarted", () => currentTrack == Beatmap.Value);
 
-            AddStep(@"Seek track to 1 second", () => musicController.SeekTo(1000));
+            AddStep(@"Seek track to 2 second", () => musicController.SeekTo(2000));
             AddStep(@"Call PrevTrack", () => musicController.PrevTrack());
-            AddAssert(@"Check if it changed to prev track'", () => currentTrack != Beatmap.Value);
+            // If the track isn't changing, check the current track's time instead
+            AddAssert(@"Check if it changed to prev track'", () => currentTrack != Beatmap.Value || currentTrack.Track.CurrentTime < 2000);
```

Comment:
Why would the track not change?

Reply:
If the available track is only one, it's basically just play the same track and restart it (both go to previous or go to next track)

![1571219587168](https://user-images.githubusercontent.com/6506864/66908590-8eba0780-f035-11e9-8c15-6edd5fb4d26a.gif)

Reply:
That needs to be split into multiple tests. As it is this test is potentially bypassing expected behaviour.

Reply:
The presence of more than one beatmap/track might not be guaranteed in a test environment, possibly we might want to ensure it by manually importing before the test run?

Reply:
Is there any testcase example that I can follow to encapsulate the beatmap list per test?

Reply:
I suppose [`ImportBeatmapTest`](https://github.com/ppy/osu/blob/master/osu.Game.Tests/Beatmaps/IO/ImportBeatmapTest.cs) has the closest thing to an example of an import in tests, although I'm not entirely certain whether we want to go down that route or not.